packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -350,24 +350,24 @@ module.exports.subscriptionsInvoicePIExpandedValidator = isA
 
 // This is subhub's perspective on an active subscription
 module.exports.subscriptionsSubscriptionValidator = isA.object({
```

Comment:
Rather than creating yet-another-validator-schema, I modified the existing subscriptionsSubscriptionValidator to make all fields but the subset for the IAP subscriptions optional.

In general Joi validation seems redundant to our type system, so I don’t think these validators are adding much in the first place, but I’m happy to create a new Joi schema if preferred and rename this one (or open to other suggestions).